docs/migration-guide/to-15.md

```diff
@@ -165,3 +165,179 @@ If you would like to keep the previous behavior, you should change your config t
   ]
 }
 ```
+
+## List of Deprecated Rules
```

Comment:
[suggest] Sorry for the minor suggestion, but how about keeping header consistency?

```suggestion
## List of deprecated rules
```